tensorflow/contrib/ios_examples/simple/tf_ios_makefile_example.xcodeproj/project.pbxproj

```diff
@@ -302,14 +306,18 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.google.TF-Test";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SEPARATE_STRIP = NO;
 			};
 			name = Debug;
 		};
 		591157B41CF4011D00C31E3A /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_DEBUG_INFORMATION_LEVEL = default;
```

Comment:
As an update, I tried disabling RTTI and the full protobuf failed with multiple errors, as did TensorFlow when I tried to compile against it. It's still worth investigating since this could be a big size win, but I'll leave that for a later change.